pages/form/edit/[formName].js

```diff
@@ -483,6 +487,38 @@ const FormEditor = () => {
                         },
                     ]}
                 />
+
+                <Card 
+                    display='grid'
+                    gridTemplateColumns='1fr'
+                    gridColumnGap={theme.space[5]}
```

Comment:
I don't think grid-column-gap is necessary.

Reply:
you're right it's not necessary, will remove it.